src/components/CheckListContainer.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import CheckList from "./CheckList";
+import { Container, Row } from "react-bootstrap";
+import "./CheckList.css";
+
+const ChecklistContainer = ({ data }) => {
+  console.log(data);
+  return (
+    <Container>
+      <Row className="py-5">
+        {data.map((info) => (
+          <CheckList
```

Comment:
there need to be a key here for each item or else it will show up a warning message so please add a kay with a value of the id of the info